site/en/blog/fledge-service-overview/index.md

```diff
@@ -153,23 +155,15 @@ FLEDGE for Chrome currently allows a
 for the Key/Value service, which will need to be migrated to TEEs in the future.
 The BYOS model is not in scope for [Bidding and Auction services](https://github.com/privacysandbox/fledge-docs/blob/main/bidding_auction_services_api.md).
 
-To ease the migration from the BYOS model, we're providing new [open source
-APIs, documentation, server implementation, and explainers](/blog/open-sourcing-fledge-key-value-service/)
-with additional capabilities beyond those already proposed. The APIs intend to
-allow custom scripts and adtech-defined code which can be run on TEEs.
+To ease the migration from the BYOS model, we're providing new [open source APIs, documentation, server implementation, and explainers](/blog/open-sourcing-fledge-key-value-service/) for the FLEDGE Key/Value service with additional capabilities beyond those already proposed. The APIs intend to allow custom scripts and adtech-defined code which can be run on TEEs.
 
 Chrome and Android will be open sourcing a Key/Value service so that adtech
 platforms can monitor development and potentially contribute to the data
 plane’s codebase.
 
 #### Timeline
 
-Adtech platforms who have implemented the BYOS model can consider migrating to
-a TEE-based Key/Value service implementation while FLEDGE is still in
-development. 
-
-We aim to have an initial TEE-based version of the Key/Value service available
-for experimentation by the end of 2022.
+Adtech platforms who have implemented the BYOS model can consider migrating to a TEE-based Key/Value service implementation while FLEDGE is still in development. In the long-term, adtechs will need to use the open source FLEDGE Key/Value services running in trusted execution environments (TEEs) for retrieving real-time data. To ensure that the ecosystem has sufficient time to test, we don’t expect to require the use of the open-source Key/Value services or TEEs until sometime after third-party cookie deprecation. We will provide substantial notice for developers to begin testing and adoption before this transition takes place.
```

Comment:
This was discussed for a long time, and I don't feel comfortable making this change at 
```suggestion
Adtech platforms who have implemented the BYOS model can consider migrating to
a TEE-based Key/Value service implementation while FLEDGE is still in
development.

In the future, adtechs will need to use the open source FLEDGE Key/Value services
running in trusted execution environments (TEEs) to retrieve real-time data.
To ensure that the ecosystem has sufficient time to test, we don’t expect to
require the use of the open-source Key/Value services or TEEs until sometime
after third-party cookie deprecation. We will provide substantial notice for
developers to begin testing and adoption before this transition takes place.
```

Reply:
Synced offline